rviz_default_plugins/include/rviz_default_plugins/displays/twist/twist_display.hpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2023, Open Source Robotics Foundation, Inc.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *     * Redistributions of source code must retain the above copyright
+ *       notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *     * Neither the name of the Willow Garage, Inc. nor the names of its
+ *       contributors may be used to endorse or promote products derived from
+ *       this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+
+#ifndef RVIZ_DEFAULT_PLUGINS__DISPLAYS__TWIST__TWIST_DISPLAY_HPP_
+#define RVIZ_DEFAULT_PLUGINS__DISPLAYS__TWIST__TWIST_DISPLAY_HPP_
+
+#include <memory>
+
+#include "rviz_default_plugins/displays/screw/screw_display.hpp"
+
+#include <geometry_msgs/msg/twist_stamped.hpp>
+
+namespace rviz_default_plugins
+{
+namespace displays
+{
+class TwistStampedDisplay : public ScrewDisplay<geometry_msgs::msg::TwistStamped>
+{
+  Q_OBJECT
+
+  // Function to handle an incoming ROS message.
+  void processMessage(geometry_msgs::msg::TwistStamped::ConstSharedPtr msg) override
+  {
+    processMessagePrivate(msg->header, msg->twist.linear, msg->twist.angular);
+  }
```

Comment:
This is a small nit, but since we have to have a `.cpp` for this class anyway, I'd prefer to move this implementation there.  It will just make compilation for downstream consumers a bit faster.

Reply:
https://github.com/ros2/rviz/pull/991/commits/67bfdf8572ae91f02cba315ca2c349ee487b7a0d

Reply:
I think this one is still open.